### PR TITLE
DCS-664 Fixing sending reminders

### PR DESCRIPTION
**Pull request by andrewrlee**
Refactoring lead to wrong import being used. Fixed and moved area to typescript (which would have caught the issue)

### CODE REVIEW

File: job/sendReminders.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+/*
+ * Do appinsights first as it does some magic instrumentation work, i.e. it affects other 'require's
+ * In particular, applicationinsights automatically collects bunyan logs
+ */
+import { initialiseAppInsights, buildAppInsightsClient } from '../server/utils/azure-appinsights'
+
+initialiseAppInsights()
+
+import StatementsClient from '../server/data/statementsClient'
+import IncidentClient from '../server/data/incidentClient'
```
Comment (andrewrlee) [inline job/sendReminders.ts:11]: the actual bug was here: was pulling in draftReportClient instead

File: server/utils/nunjucksSetup.ts [hunk 2]
```diff
@@ -16,7 +17,7 @@ type Error = {
   text: string
 }
 
-export default function (app, path) {
+export default function configureNunjucks(app: Express.Application): void {
```
Comment (andrewrlee) [inline server/utils/nunjucksSetup.ts:20]: fixing some type issues here and noticed path didn't need to be injected as a dependency

**Review by JamieDraperUK — APPROVED** [review]
LGTM

**Review by bell-pa — APPROVED** [review]
LGTM

**Pull request merged**